api/v1alpha1/types_etcd.go

```diff
@@ -381,8 +383,15 @@ type EtcdStatus struct {
 	// +optional
 	ServiceName *string `json:"serviceName,omitempty"`
 	// LastError represents the last occurred error.
+	// Deprecated: Use LastErrors instead.
```

Comment:
It's generally not a good idea to introduce a breaking change in a publicly exposed API without prior warning to its users. There may be users who actually check `LastError`, and we wouldn't want them to suffer from a sudden removal of the field altogether. Hence, we add a deprecation note, and remove it after 2-3 releases, so that users have sufficient time to switch to using `LastErrors` field instead.

Reply:
Thanks.

Reply:
Could we create an issue to track and plan the removal of the deprecated field `LastError` in the near future?

Reply:
Sure. I have added this to https://github.com/gardener/etcd-druid/issues/645, so we can track it in that issue itself.